plugins/GSdx/GLLoader.cpp

```diff
@@ -367,7 +367,12 @@ namespace GLLoader {
 #ifdef _WIN32
 		if (status) {
 			if (fglrx_buggy_driver) {
-				fprintf(stderr, "OpenGL renderer is slow on AMD GPU due to inefficient driver. Sorry.\n");
+				fprintf(stderr, "The \"OpenGL Hardware\" renderer is slow on AMD GPUs due to an inefficient driver. Check out the links below for further information.\n"
+					"https://community.amd.com/message/2756964\n"
+					"https://community.amd.com/thread/205702\n"
+					"Note: Due to an AMD GPU OpenGL driver issue, setting Blending Unit Accuracy to \"None\" can cause an application or system crash.\n"
+					"Keep Blending Unit Accuracy set to at least the default \"Basic\".\n"
```

Comment:
``to at least the default \"Basic\"`` **->** ``to at least the default \"Basic\" level.``